### PR TITLE
Feature/enable Get logtypes and logs endpoint

### LINKED ISSUE #280: capturing ADB logcat when test is running
I want to have adb logs for the test app so that dev can have look at adb logs when there is failure. This Functionality is already implemented in java client.

### PR DESCRIPTION
**Pull request by akinsolb**
## Change list
AppiumCommands.cs

Please provide briefly described change list which are you going to propose. 
 
## Types of changes

What types of changes are you proposing/introducing to .NET client?
_Put an `x` in the boxes that apply_

- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] New feature (non-breaking change which adds value to the project)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Details
Add support for accessing log types and logs - resolves #280 #289

### CODE REVIEW

File: src/Appium.Net/Appium/AppiumCommand.cs [hunk 1]
```diff
@@ -194,6 +194,9 @@ public class AppiumCommand
             #region Compare Images
             new AppiumCommand(CommandInfo.PostCommand, AppiumDriverCommand.CompareImages, "/session/{sessionId}/appium/compare_images" ),
             #endregion
+
+            new AppiumCommand(CommandInfo.GetCommand, DriverCommand.GetAvailableLogTypes, "session/{sessionId}/log/types"),
```
Comment (mykola-mokhnach) [inline src/Appium.Net/Appium/AppiumCommand.cs]: I thought these endpoints were already present for selenium

Comment (manoj9788) [reply src/Appium.Net/Appium/AppiumCommand.cs]: I think it is available - let me check

Comment (akinsolb) [reply src/Appium.Net/Appium/AppiumCommand.cs]: I cannot remember off the top of my head why I added that, they definitely exist in selenium, but I think maybe the CommandInfo value differs to appium. I'll run the tests without the entries in a moment. Will confirm after. In other words, don't merge yet :)

Comment (manoj9788) [reply src/Appium.Net/Appium/AppiumCommand.cs]: Thanks @akinsolb

Comment (akinsolb) [reply src/Appium.Net/Appium/AppiumCommand.cs]: It needs to be added because it's not in the W3CWireProtocolCommandInfoRepository

Comment (KazuCocoa) [reply src/Appium.Net/Appium/AppiumCommand.cs]: Yes, the log stuff are not in W3C.
https://github.com/appium/ruby_lib_core/blob/master/lib/appium_lib_core/common/command/w3c.rb is commands used in Appium, but selenium w3c module does not have. OSS module had them.

**Pull request merged**